qa/rpc-tests/mempool_tx_expiry.py

```diff
@@ -147,7 +186,7 @@ def run_test(self):
         assert_equal(set(self.nodes[0].getrawmempool()), set())
         sync_blocks(self.nodes)
         print "Balance after persist_shielded_2 is mined to remove from mempool: ", self.nodes[0].z_gettotalbalance()
-
+        
```

Comment:
Still one missed, but it seems that every time one is fixed, another pops up. So I'm not gonna jinx it by trying to eliminate this one...